docs/user/reference/metrics/timing_distribution.md

```diff
@@ -276,17 +276,94 @@ void onPageLoaded(Event e) {
 ```
 
 </div>
-<div data-lang="Swift" class="tab">
+<div data-lang="Swift" class="tab"></div>
+<div data-lang="Python" class="tab">
 
-```Swift
-import Glean
+```Python
+from glean import load_metrics
+metrics = load_metrics("metrics.yaml")
 
-func onPageLoaded() {
-    Pages.pageLoad.accumulateSamples(samples)
+class PageHandler:
+    def on_page_loaded(self, event):
+        metrics.pages.page_load.accumulate_samples(samples)
+```
+
+</div>
+<div data-lang="Rust" class="tab">
+
+This API is not currently exposed in Rust, see [Bug 1829745](https://bugzilla.mozilla.org/show_bug.cgi?id=1829745).
```

Comment:
Are you still planning on updating this since we determined it is now implemented for the RLB?